doc/tutorials/python-kdb.md

```diff
@@ -4,6 +4,22 @@
 
 When programming in Python it is possible to access the kdb database, changing values of existing keys, adding and deleting keys and a few other things.
 
+## Installation
+
+Either [build](https://www.libelektra.org/bindings/swig_python) the package or install from a repository.
+
+### Alpine Linux
+
+The [python bindings package](https://pkgs.alpinelinux.org/packages?name=py3-elektra&branch=edge&repo=testing) is only available in the testing repository (as of 2019-04-29).
+
+You'll have to use `alpine:edge`
```

Comment:
Why "have to use"? Does this statement belong to the previous paragraph?

Please also use colon (:) at the end.

Reply:
I was under the impression that edge repositories could just be used by `alpine:edge`, but that doesn't seem to be the case.

However with these exact steps and regular alpine, the installation of `py3-elektra` and `elektra-python` fails because they need python3.7, and the regular repositories only have python3.6.

So I added additional instructions for regular alpine.